effekt/shared/src/main/scala/effekt/core/Transformer.scala

```diff
@@ -611,7 +611,7 @@ object Transformer extends Phase[Typechecked, CoreTransformed] {
 
     // create joinpoint
     val params = patterns.flatMap { case (sc, p) => boundInPattern(p) } ++ guards.flatMap(boundInGuard)
-    val joinpoint = Context.bind(TmpBlock(), BlockLit(Nil, Nil, params, Nil, body))
+    val joinpoint = Context.bind(TmpBlock("joinpoint"), BlockLit(Nil, Nil, params, Nil, body))
```

Comment:
Typically, I would use `k` for joinpoints. But we can also use `j` if `k` is used too often.